src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+public class Event extends Task {
+
+    protected String at;
+
+    public Event(String description, String at) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
I believe this is still needed because if I included a print message function in my super. If I use super here the message will be printed out without storing the deadline and task. Thank u for the suggestion nonetheless!